nav2_navfn_planner/src/navfn.cpp

```diff
@@ -667,6 +672,10 @@ NavFn::propNavFnAstar(int cycles)
 
   // do main cycle
   for (; cycle < cycles; cycle++) {  // go for this many cycles, unless interrupted
+    if (cycle % cancel_check_interval == 0 && cancelChecker()) {
+      throw nav2_core::PlannerCancelled("Planner was cancelled");
```

Comment:
I don't see where this exception is caught specially to handle this condition now that its created

Reply:
It should be in the catch list and if caught, do what we do in the cancel function: https://github.com/ros-planning/navigation2/blob/9e0e1f6b785d89cebd876dfbe1fbed7bea8c644f/nav2_planner/src/planner_server.cpp#L298

Reply:
Done.